PatternPal/PatternPal.LoggingServer/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
```

Comment:
Why don't you use `net7.0` here? That's the version we use in the rest of the project, and I'd prefer to keep that version one and the same throughout the project, with exception of the VS extension of course (as it requires .NET Framework).